### PR TITLE
Fix hub.docker.com URL

### PR DESCRIPTION
**Pull request by mlovci**
I had changed the url to reference /u/ipfs in a commit that is now skipped but it should really be /u/jbenet

Now the docker hub url is hub.docker.com/u/jbenet/go-ipfs

### CODE REVIEW

**Comment by mlovci** [comment]
Build shouldn't be failing, I didn't change any code. Only docs.

**Comment by jbenet** [comment]
@mlovci sorry that's our tests being silly.

File: README.md [hunk 1]
```diff
@@ -104,10 +104,10 @@ Start a container running ipfs and expose ports 4001, 5001 and 8080:
...
     docker run -d --name ipfs_host -v $ipfs_staging:/export -v $ipfs_data:/root/.go-ipfs -p 8080:8080 -p 4001:4001 -p 5001:5001 jbenet/go-ipfs:latest
     
-Wait for ipfs to start. ipfs is running when you  see "Gateway 
-(readonly) server 
-listening on /ip4/0.0.0.0/tcp/8080" in the container logs (`docker logs -f 
-ipfs_host`)
+Wait for ipfs to start. ipfs is running when you see: 
+
+    Gateway (readonly) server 
+    listening on /ip4/0.0.0.0/tcp/8080" in the container logs (`docker logs -f ipfs_host`)
```
Comment (jbenet) [inline README.md]:
```
    listening on /ip4/0.0.0.0/tcp/8080

in the container logs (`docker logs -f ipfs_host`)
```

(formatting bug)

**Comment by mlovci** [comment]
darn you Travis :facepunch:

File: README.md (hunk 1 above)
Comment (mlovci) [inline README.md]: apologies. fixing

**Comment by jbenet** [comment]
@mlovci wont let me merge-- maybe this conflicts with your last PR? try:

```
git remote add upstream https://github.com/ipfs/go-ipfs
git fetch upstream
git rebase upstream/master
git push origin master -f
```

**Comment by mlovci** [comment]
Hm, seems that when I rebased it broke the ability to merge back to your repo. I'll do it the sloppy way and keep the commit with the wrong hub.docker... URL

**Pull request merged**